src/core.js

```diff
@@ -2002,10 +2027,8 @@ define([
                 properties: this.getMainProperties(),
                 help: {
                     title: "Basic",
-                    text: "<p>The <strong>Question ID</strong> is an internal identifier for a question. " +
-                        "It does not appear on the phone. It is the name of the question in data exports.</p>" +
-                        "<p>The <strong>Label</strong> is text that appears in the application. " +
-                        "This text will not appear in data exports.</p> ",
+                    text: "<p>The <strong>Label</strong> is text that appears in the application. " +
```

Comment:
For a later PR these help texts seem strange. I think they should be on the question property and not on the accordion